src/test/java/bio/terra/endpoints/CLITestingConfig.java

```diff
@@ -1,14 +1,46 @@
 package bio.terra.endpoints;
 
 public final class CLITestingConfig {
-    //public static final String dataRepoURL = "http://localhost:8080/";
-    //public static final String clientSecretsFilePath = null;
-    //public static final String dataRepoURL = "https://jade.datarepo-dev.broadinstitute.org/";
-    //public static final String clientSecretsFilePath = "/tmp/jadecli_client_secret.json";
-    public static final String dataRepoURL = "https://jade-mm.datarepo-dev.broadinstitute.org/";
-    public static final String clientSecretsFilePath = "/tmp/jadecli_client_secret.json";
+    private static CLITestingConfig theConfig;
+    private String dataRepoURL;
+    private String clientSecretsFilePath;
+    private String billingAccount;
 
     public static final String dirName = "src/test/resources/CLICommandTests/";
+    public static final String testProfileName = "jadecli_test_profile";
 
-    private CLITestingConfig() { }
+    private CLITestingConfig() {
+        dataRepoURL = System.getenv("JADECLI_TEST_DATAREPO_URL");
+        if (dataRepoURL == null) {
+            dataRepoURL = "http://localhost:8080/";
+        }
+
+        // Secrets default is null; default is supplied in the main code
+        clientSecretsFilePath = System.getenv("JADECLI_TEST_CLIENT_SECRETS");
+
+        // Billing account is required
+        billingAccount = System.getenv("JADECLI_TEST_BILLING_ACCOUNT");
```

Comment:
This environment variable may need to be added to Travis.

Reply:
Yep! Good catch. I updated it in the settings.